### ISSUE TITLE
9anime Down 

### ISSUE THREAD

**Issue by andystyle09** [DxCx/plugin.video.9anime#189]
Good evening all 9anime on firestick seems to be down unable to find eligible sources. Anyone has a fix? Thank you

**Comment by kiryu77** [comment]
It's happen to me too. I'm unable to load a single video.

**Comment by player7000000** [comment]
Same here. It usually fixes by itself. It should be fixed in a couple of hours. The 9anime website is working so it's not the websites fault

**Comment by JaeFr0st** [comment]
I was getting into Prison School when this happened lol. player7000000 is correct. 9anime from time to time has this issue and everything goes back to normal after some time. Being that the add-on is not the issue you can close this issue out.

**Comment by JaeFr0st** [comment]
I had to delete my comment from earlier. After doing some research it seems like we will have to wait until DxCx makes an update. This is an duplicate issue of #151.

**Comment by G1-Primus** [comment]
Thank god I found this post I have spent like 5 hours trying to fix this. Does it keep saying "couldn't find eligible sources" for everyone else too?

**Comment by JaeFr0st** [comment]
Yes! Everyone will be seeing the same error. It has to be fixed by DxCx. Hopefully he will have an update put out soon.

**Comment by player7000000** [comment]
Hopefully it's out before Monday. I need my Dragon Ball Super! Haha

**Comment by JaeFr0st** [comment]
Haha! Same here.

**Comment by JoshToshiro** [comment]
I was having the same issue. I’m just trying to binge watch fairy tail and this morning I had no luck . Hope it gets fixed quick.

**Comment by pjcanales** [comment]
Don't worry everyone. soon as he's see all the comment on how many of us uses his add-on is he'll be sure to fix it as soon as possible.

**Comment by Prometheusx-git** [comment]
9anime has changed their api.
Now they use ts from data ts, extra para "_", episode-id and server-id again.

**Comment by pjcanales** [comment]
Ok what's the difference between this add-on and your unofficial 9anime addon?

**Comment by here4thecomments** [comment]
So is there an update it a new add-on we need to get back on 9anime? I'm like a junky w/o heroin here!!! Lol but seriously I need it back in my life asap!!!!

**Comment by here4thecomments** [comment]
So is there an update or a new add-on we need to get back on 9anime? I'm like a junky w/o heroin here!!! Lol but seriously I need it back in my life asap!!!!

**Comment by yogendrasinghx** [comment]
What a draaagg!!!

**Comment by DxCx** [comment]
Hey all, rough week sry to keep you waiting, ill probably fix it tomorrow :)

**Comment by JaeFr0st** [comment]
It’s all good @DxCx. Cheers 🍻

**Comment by player7000000** [comment]
@DxCx thanks! We appreciate it but no rush. We know life happens. Wishing you the best! 😬

**Comment by JoshToshiro** [comment]
@DxCx yeah don’t sweat it dude. Do what you need to do and take care of it when you can 👍

**Comment by andystyle09** [comment]
@DxCx  thanks for looking into it I’m just glad you saw it now I know it’s gonna be fixed eventually which I’m happy about thank you for your hard work !

**Comment by here4thecomments** [comment]
@DxCx ty for the heads up!

**Comment by jimboooooo** [comment]
Mine is still saying the same thing. Am I suppose to update it or something?

**Comment by andystyle09** [comment]
Mine  is working perfectly fine now thank you just do a manual update if yours is not

**Comment by jimboooooo** [comment]
I did @andystyle09 and nothing... it’s still just version 0.0.27. I’ll give it another try tomorrow I guess 🤷🏽‍♂️

**Comment by tinymovingparts** [comment]
@jimboooooo this add on is different for its auto updates. I should update fine by tomorrow on its own if you have auto update on. But you can manual update this by updating the 9anime repository on your device

**Comment by JaeFr0st** [comment]
@jimboooooo best option is to download git browser add-on for kodi. Search by git user name DxCx and it will show all of DxCx’s work. You’ll see 9anime latest update there. To be honest Git Browser is the best thing to happen to kodi. Makes it super easy to add add-ons without having to type the source and go through a lengthy process to install.

**Comment by JaeFr0st** [comment]
@jimboooooo you can also wait until it updates itself tomorrow as @tinymovingparts said in his response.

**Comment by jimboooooo** [comment]
@jaefr0st I’m about to look into that right now... hopefully I can finesse that... and @DxCx good job... best anime app out by FARRR 👍🏽

**Comment by tinymovingparts** [comment]
@jimboooooo if youve never manual updated the repository before all you have to do is go to install from repository, find the 9anime repository, right click and select check for updates and it will update everything from there including the addon

**Comment by JaeFr0st** [comment]
@jimboooooo no problem. @tinymovingparts answer is correct too.

**Comment by jimboooooo** [comment]
The Git browser worked like a charm... that’s a lot bro 👍🏽🔥 @JaeFr0st

**Comment by JaeFr0st** [comment]
No problem @jimboooooo. I wish everyone who made addons was on GitHub. Git Browser is truly amazing and so simple. I hate having to go through a lot of steps such as add a source, install from zip and then install from repository. With Git Browser you just find their GitHub name and just click what ever zip you want to install and BAM! You’re good to go.

**Comment by jimboooooo** [comment]
Thanks*

**Comment by jimboooooo** [comment]
Seriously dude.... git browser is like the best kept secret

**Comment by DxCx** [comment]
Well, yeah, fix was released on version 0.0.28, auto update should update it :)

**Comment by JoshToshiro** [comment]
@JaeFr0st how do I get the indigo tool or where do I find it? 😅

**Comment by JoshToshiro** [comment]
@JaeFr0st so I can download the Git browser

**Comment by JoshToshiro** [comment]
@DxCx you are the GOAT 🔥, thanks

**Comment by JaeFr0st** [comment]
@JoshToshiro  just follow this guide 😀http://whyingo.org/how-to-install-indigo-add-on-wizard-kodi-17-1-krypton/

**Comment by JoshToshiro** [comment]
@JaeFr0st thanks a bunch bro 😁 about to do that right now

**Comment by JaeFr0st** [comment]
@JoshToshiro follow this guide 😀 https://www.tvaddons.co/fusion-kodi-krypton/ disregard previous message if you still some how manage to see it still. It’s been deleted by me as it was inaccurate.

**Comment by JoshToshiro** [comment]
@JaeFr0st oh ok, no problem, thanks again.

**Comment by JaeFr0st** [comment]
@JoshToshiro  actually give me a few. I’m going to have to re-edit that comment. Don’t follow that guide.

**Comment by JoshToshiro** [comment]
@JaeFr0st lol alright, take your time.

**Comment by JaeFr0st** [comment]
Okay commmet has been updated. If you have any issues just let me know

**Comment by JoshToshiro** [comment]
@JaeFr0st , ok, I appreciate the tip 👍

**Comment by player7000000** [comment]
Hey guys I'm still having the no sources thing for a couple of animes. Like Black Clover. I can't play episode 16

**Comment by player7000000** [comment]
I'm already on the newest update

**Comment by JaeFr0st** [comment]
@player7000000 maybe it wasn’t uploaded properly and that’s why there’s no source for it or maybe it wasn’t uploaded at all. Not sure. Also just a heads up and some info I wanted to share but I saw a post that 9Anime posted on twitter and reddit but both have been removed by reddit and Twitter due to a copy right claim. Rumor is that 9Anime probably will be shut down due to some copyright claim. Not sure what’s going on moment. Nothing has been posted from them lately.

**Comment by cavebeast86** [comment]
Mine updated but same issue works on pc and pi but not on fire stick.  Also on pc I've been getting no host found on a lot of popular stuff for example anything past so 4 on black clover

**Comment by tinymovingparts** [comment]
The addon is working fine with the 0.0.28 update. Unfortunately the issue with not finding a host has to do with the actual host site and not with the addon itself. The issues are commonly recent and repetitive with the dubs of newer shows probably due to getting pulled from the host once put up.

**Comment by cavebeast86** [comment]
I've been having the issue for about 2 weeks on my android devices like I said pi and pc still work fine.

**Comment by tinymovingparts** [comment]
your comment said "Also on pc I've been getting no host found on a lot of popular stuff for example anything past so 4 on black clover" can newer dubbed stuff like black clover ep 5 work on your pc through the addon?

**Comment by cavebeast86** [comment]
I did still the same I uninstall the add-on and repo and make sure to update both on reinstall it works but won't go past the main directory so it won't let me actually see any content just the new and popular folders
